app/controllers/application_controller.rb

```diff
@@ -5,6 +5,143 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+
+    enable :sessions unless test?
+    set :session_secret, "secret"
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :"tweets/create_tweet"
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets/new' do
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.